android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -907,16 +915,26 @@ private WritableArray getAudioTrackInfo() {
         TrackGroupArray groups = info.getTrackGroups(index);
         for (int i = 0; i < groups.length; ++i) {
             Format format = groups.get(i).getFormat(0);
+            String language = format.language != null ? format.language : "";
+            String languageCode = this.getIso2(language);
+
             WritableMap audioTrack = Arguments.createMap();
             audioTrack.putInt("index", i);
             audioTrack.putString("title", format.id != null ? format.id : "");
             audioTrack.putString("type", format.sampleMimeType);
-            audioTrack.putString("language", format.language != null ? format.language : "");
+            audioTrack.putString("language", languageCode);
             audioTracks.pushMap(audioTrack);
         }
         return audioTracks;
     }
 
+    private String getIso2(String iso3) {
```

Comment:
I think it'll make more sense to have the accronym namings similar to standard library (all upper case `getISOLanguages`)

`getIso2` -> `getISO2`